gap_fit_module.f95

```diff
@@ -673,17 +749,22 @@ subroutine fit_n_from_xyz(this)
     this%n_force = 0
     this%n_virial = 0
     this%n_hessian = 0
+    this%n_local_property = 0
+    n_last = 0
 
     do n_con = 1, this%n_frame
-
-       has_ener = get_value(this%at(n_con)%params,this%energy_parameter_name,ener)
-       has_force = assign_pointer(this%at(n_con),this%force_parameter_name, f)
-       has_virial = get_value(this%at(n_con)%params,this%virial_parameter_name,virial)
-       has_stress_voigt = get_value(this%at(n_con)%params,this%stress_parameter_name,stress_voigt)
-       has_stress_3_3 = get_value(this%at(n_con)%params,this%stress_parameter_name,stress_3_3)
-       has_hessian = get_value(this%at(n_con)%params,"n_"//this%hessian_parameter_name,n_hessian)
-       has_local_property = assign_pointer(this%at(n_con),this%local_property_parameter_name, local_property)
-       has_force_mask = assign_pointer(this%at(n_con),trim(this%force_mask_parameter_name),force_mask)
+       if (do_filter_tasks) then
+          if (this%task_manager%tasks(n_con)%worker_id /= this%task_manager%my_worker_id) cycle
+       end if
+
+       has_ener = get_value(this%at(n_con)%params, this%energy_parameter_name, ener)
```

Comment:
more whitespace changes?

Reply:
Same everywhere. Oversight when adding changes. Can be removed.